PlayerCore/player model/VRM/VRMResponseModel.swift

```diff
@@ -5,19 +5,33 @@ import Foundation
 
 public extension VRMCore {
     
+    public struct ID<ParentType>: Hashable {
+        public let value: UUID
```

Comment:
I would rather drop this until we really need this value :)

Reply:
Done